server/webserver.go

```diff
@@ -677,7 +677,23 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 
 	mux.HandleFunc("/withdrawFees", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth != nil {
-			tx, err := s.LivepeerNode.Eth.WithdrawFees()
+			if err := r.ParseForm(); err != nil {
```

Comment:
Since we're updating this handler it could be a good opportunity for a small refactor which also improves testability of handlers now that we have additional helpers in the codebase for handling form values.

See this example [hander definition](https://github.com/livepeer/go-livepeer/blob/81a42873663bd34cbba007cb11a5554f84404f92/server/handlers.go#L92), [attaching it to the mux](https://github.com/livepeer/go-livepeer/blob/81a42873663bd34cbba007cb11a5554f84404f92/server/webserver.go#L1175) and its associated [tests](https://github.com/livepeer/go-livepeer/blob/81a42873663bd34cbba007cb11a5554f84404f92/server/handlers_test.go#L188).

An example of what attaching this handler to the mux could look like:

```
mux.Handle("/withdrawFees", mustHaveFormParams(withdrawFeesHandler(s.LivepeerNode.Eth), "amount")
```

WDYT?

Reply:
Good idea. I'll refactor all the handlers.